### ISSUE TITLE
trust no one, not even your self

### ISSUE THREAD

**Issue by safarirat** [mozilla/multi-account-containers#1499]
https://imgur.com/a/enWFn7y
Remove the note that you install the addon at your own risk.

**Comment by jonathanKingston** [comment]
This will be fixed soon when we do a new release (sometime this week I believe).
/cc @maxxcrawford

**Comment by chujunlu** [comment]
Hi @maxxcrawford , I'm Chujun, an Outreachy applicant. I'm interested in privacy and security related projects so I'd like to see if I can contribute to this one. I didn't see you or Luke in the IRC channel and I wasn't sure if we have to be in the channel at the same time to receive messages. So excuse me that I asked questions here.

1.I'm reading test files to familiarize myself with the code base. Is it possible to run a test file in a browser debugger? I tried `node --inspect-brk test/features/assignment.test.js` but it said `describe` is undefined.

2.I'm also reading merged pull requests. I tried to run a certain part of the code in debugger following docs here https://extensionworkshop.com/documentation/develop/debugging/. I'm curious is that the usual way to go about debugging?

Thanks for your guidance!
